F21/rishith-cmykt/README

```diff
@@ -1 +1,23 @@
 Your program for the code review lab goes in this directory
+#ENGLSIH LANGUAGE
+y=input("Enter any word in uppercase letters : ")
+c=y.lower()
+print("Lower case letters : ",c)
+#IRISH LANGUAGE
+r = input("Enter any word in irish capital letters : ")
+if (r[0] == 'n' or r[0] == 't') and (r[1] == 'A' or r[1] == 'E' or r[1] == 'I' or r[1] == 'O' or r[1] == 'U' or r[1] == 'Á' or r[1] == 'É' or r[1] =='Í' or r[1] =='Ó' or r[1] == 'Ú'):
+    print(r[0].rstrip(),'-',r[1:].lower())
+else:
+    print(r.lower())
+#GREEK language
+O=input("Enter any word in greek capital letters: ")
+c=O.lower()
+print(c)
+#TURKISH language
+def _init_(self,word,language):
```

Comment:
Is there a reason that for turkish you are using a function opposed to user input like you did previously?